Missing attributes inside Button distribution

# 🐛 Bug Report 

I'm working in our company monorepo and was adding some type checking to it. I initially thought it must be user error, but I dug a little deeper and noticed the types in the button distribution are missing imports. I can see they should be defined in (as they are in the repository), but in my node_modules they are not.

Here's what I'm seeing inside node_modules/@react-aria/button/dist/types.d.ts:
```
import { ElementType, HTMLAttributes, RefObject } from "react";
import { AriaButtonProps, AriaToggleButtonProps } from "@react-types/button";
...
export function useButton(props: AriaButtonProps<'a'>, ref: RefObject<HTMLAnchorElement>): ButtonAria<AnchorHTMLAttributes<HTMLAnchorElement>>;
```

tsc fails because of missing ```AnchorHTMLAttributes```, this pattern repeats for```ButtonHTMLAttributes``` and also```InputHTMLAttributes```.


## 🤔 Expected Behavior

Button.d.ts should be importing the missing attributes (my editor suggested the following):
```import {AnchorHTMLAttributes, ButtonHTMLAttributes, ElementType, HTMLAttributes, InputHTMLAttributes, RefObject } from "react";```

## 😯 Current Behavior

They don't exist, it is only happening inside Button which is interesting.

The error is self-explanatory but: ```../../node_modules/@react-aria/button/dist/types.d.ts:11:108 - error TS2304: Cannot find name 'ButtonHTMLAttributes'.```

## 💁 Possible Solution

Whatever is generating the import needs adjusting.

## 🔦 Context

Working with tsc. 

## 💻 Code Sample

I double and tripled checked through this just to be certain.

I found a way to show the problem:
If we download the tarball from NPM: https://registry.npmjs.org/@react-aria/button/-/button-3.3.0.tgz

Open up dist/types.d.ts, you'll see the missing imports.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-aria   | ^3.1.1
| Browser          | N/A
| Operating System | Mac

--- a/patches/@parcel+transformer-typescript-types+2.0.0-nightly.359.patch
+++ b/patches/@parcel+transformer-typescript-types+2.0.0-nightly.359.patch
@@ -1,5 +1,5 @@
 diff --git a/node_modules/@parcel/transformer-typescript-types/lib/TSModule.js b/node_modules/@parcel/transformer-typescript-types/lib/TSModule.js
-index 981ca7b..6cd9ebb 100644
+index 981ca7b..c02862b 100644
 --- a/node_modules/@parcel/transformer-typescript-types/lib/TSModule.js
 +++ b/node_modules/@parcel/transformer-typescript-types/lib/TSModule.js
 @@ -33,7 +33,7 @@ class TSModule {
@@ -11,8 +11,19 @@ index 981ca7b..6cd9ebb 100644
      }
    }
  
+@@ -52,7 +52,9 @@ class TSModule {
+   }
+ 
+   addLocal(name, node) {
+-    this.bindings.set(name, node);
++    const bindings = this.bindings.get(name) || new Set();
++    this.bindings.set(name, bindings);
++    bindings.add(node);
+ 
+     if (name !== 'default') {
+       this.names.set(name, name);
 diff --git a/node_modules/@parcel/transformer-typescript-types/lib/TSModuleGraph.js b/node_modules/@parcel/transformer-typescript-types/lib/TSModuleGraph.js
-index f346712..830f180 100644
+index f346712..18a476e 100644
 --- a/node_modules/@parcel/transformer-typescript-types/lib/TSModuleGraph.js
 +++ b/node_modules/@parcel/transformer-typescript-types/lib/TSModuleGraph.js
 @@ -48,17 +48,13 @@ class TSModuleGraph {
@@ -37,7 +48,23 @@ index f346712..830f180 100644
      }
  
      if (module.used.has(name)) {
-@@ -130,8 +126,8 @@ class TSModuleGraph {
+@@ -81,10 +77,11 @@ class TSModuleGraph {
+       return ts.visitEachChild(node, visit, context);
+     };
+ 
+-    let node = module.bindings.get(name);
+-
+-    if (node) {
+-      ts.visitEachChild(node, visit, context);
++    let bindings = module.bindings.get(name);
++    if (bindings) {
++      for (let node of bindings) {
++        ts.visitEachChild(node, visit, context);
++      }
+     }
+   }
+ 
+@@ -130,8 +127,8 @@ class TSModuleGraph {
  
      return {
        module: m,
@@ -48,7 +75,7 @@ index f346712..830f180 100644
      };
    }
  
-@@ -223,6 +219,7 @@ class TSModuleGraph {
+@@ -223,6 +220,7 @@ class TSModuleGraph {
        exportedNames.set(e.name, e.module);
      } // Assign unique names across all modules
  
@@ -56,7 +83,7 @@ index f346712..830f180 100644
  
      for (let m of this.modules.values()) {
        for (let [orig, name] of m.names) {
-@@ -230,7 +227,13 @@ class TSModuleGraph {
+@@ -230,7 +228,13 @@ class TSModuleGraph {
            continue;
          }
  
@@ -71,7 +98,7 @@ index f346712..830f180 100644
            continue;
          }
  
-@@ -242,6 +245,37 @@ class TSModuleGraph {
+@@ -242,6 +246,37 @@ class TSModuleGraph {
        }
      }
  

